cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocConf.java

```diff
@@ -136,6 +139,11 @@ public String getNetwork() {
     return network;
   }
 
+  @Nullable
+  public String getNetworkHostProjectID() {
+    return NETWORK_HOST_PROJECT_ID;
```

Comment:
should be `return networkHostProjectId;`.

Reply:
oops! Done.